doc/apiv3-documentation.apib

```diff
@@ -3390,41 +3430,306 @@ For more details and all possible responses see the general specification of [Fo
 
     [Example Form][]
 
-## Add Watcher [/api/v3/work_packages/{work_package_id}/watchers]
++ Response 403 (application/hal+json)
 
-## Add watcher [POST]
+    Returned if the work package does not exist or the client does not have sufficient permissions to see it.
```

Comment:
Copy pasta? Here we should already have the permission to see a work package but lack the permission to edit it.